docs/content/schemas.md

```diff
@@ -450,6 +450,48 @@ When we derive a type from a schema, we try to match what is known and unknown i
 
 When overriding existing types, the dynamicity of the overridden prefix is preserved.
 
+### Supporting JSON Schema composition keywords 
+
+JSON Schema provides keywords such as `anyOf` and `allOf` to structure a complex schema. For `anyOf`, at least one subschemas must be true, and for `allOf`, all subschemas must be true. The type checker is able to identify such keywords and derive a more robust Rego type through more complex schemas. 
```

Comment:
[nit] 
```suggestion
JSON Schema provides keywords such as `anyOf` and `allOf` to structure a complex schema. For `anyOf`, at least one of the subschemas must be true, and for `allOf`, all subschemas must be true. The type checker is able to identify such keywords and derive a more robust Rego type through more complex schemas. 
```

or _"...at least one subschema must be true, ..."_

Reply:
Done (651e9803e56d4937c0e1a1b2afcc6d0aa4c6342d)